django_socio_grpc/management/commands/grpcrunaioserver.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from concurrent import futures
+from datetime import datetime
+import sys
+import errno
+import os
+import logging
+
+import grpc
+import asyncio
+from django.utils import autoreload
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from django_socio_grpc.settings import grpc_settings
+
+# logger = logging.getLogger("django_socio_grpc")
```

Comment:
put back logging as it only affect your project

Reply:
Ok